exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,15 @@
 var name = "Daniel";
 var danielsRole = "mentor";
 
+function isDanielStudentOrMentor(names){
+  if(name.toLowerCase() + "sRole" === "mentor"){
+    return "Hi, I'm Daniel, I'm a mentor";
+  } else {
+    return "Hi, I'm Daniel, I'm a student";
+  }
+}
+
+console.log(isDanielStudentOrMentor(name));
 /* 
```

Comment:
Amanda, here toLowerCase() command confused me. I think the condition isn't related to upper and lower cases. When I give it a try, it returned with
'Hi, I'm Daniel, I'm a student'
the expected result is 'Hi, I'm Daniel, I'm a mentor'

Reply:
I see I will fix it right now. Thank you for pointing it out. The toLowerCase turns Daniel into daniel.